target/linux/ath79/dts/qca9563_tplink_archer-c6-v2.dts

```diff
@@ -7,3 +7,113 @@
 	compatible = "tplink,archer-c6-v2", "qca,qca9563";
 	model = "TP-Link Archer C6 v2";
 };
+
+/ {
+
```

Comment:
There already if a root node above, so please include this there.

Reply:
ok, mistype. sorry.